Cargo.toml

```diff
@@ -62,11 +83,12 @@ default-features = false
 [profile.dev]
 opt-level = 1      # controls the `--opt-level` the compiler builds with
 debug = true       # controls whether the compiler passes `-C debuginfo`
-                   # a value of `true` is equivalent to `2`
+# a value of `true` is equivalent to `2`
 rpath = false      # controls whether the compiler passes `-C rpath`
 lto = false         # controls `-C lto` for binaries and staticlibs
 debug-assertions = true # controls whether debug assertions are enabled
-panic = "abort" # Call abort on panic https://github.com/rust-lang/rust/pull/32900
+# Disabled because of https://github.com/rust-lang/cargo/issues/7359
+#panic = "abort" # Call abort on panic https://github.com/rust-lang/rust/pull/32900
```

Comment:
Disabling this doesn't seem to have any negative side effects, but I wanted to point this out in case I'm overlooking something.